app/services/post_alerter.rb

```diff
@@ -628,7 +628,8 @@ def notify_post_users(post, notified, include_category_watchers: true, include_t
 
     each_user_in_batches(notify) do |user|
       notification_type = already_seen_user_ids.include?(user.id) ? Notification.types[:edited] : Notification.types[:posted]
-      create_notification(user, notification_type, post)
+      opts = notification_type == Notification.types[:edited] ? { display_username: post.last_editor.username } : {}
```

Comment:
I think this would be more clear as two lines:

```
opts = {}
opts[:display_username] = post.last_editor_username if notification_type == Notification.types[:edited]
```